### ISSUE TITLE
No assets on 404 page when deployed to Github Pages

### ISSUE THREAD

**Issue by Vinno97** [mkdocs/mkdocs#1283]
Hi,

It appears that the `404` page has an error when it's deployed to `Github Pages`, namely that the assets aren't linked properly.

---

On normal pages, the URLs are relative to the current page and always point to the proper assets. For example:
```
../assets/stylesheets/application-b1a1975878.css
```
May result in:
```
https://<username>.github.io/<repository_name>/assets/stylesheets/application-b1a1975878.css
```

---

On the 404 page, the assets have a static link:

```
/assets/stylesheets/application-b1a1975878.css
```
This will then result in:
```
https://<username>.github.io/assets/stylesheets/application-b1a1975878.css
```
and case a `404` error for this resource, which causes the page to be unstyled static text.

---

I don't know a proper fix for this, except adding the name of the repository to the path when it's deployed to Github Pages.
<br />

Thank you,

Vincent

**Comment by waylan** [comment]
What is your [`site_url`][1] set to in your `mkdocs.yml` config file? It should be:

```yaml
site_url: https://<username>.github.io/<repository_name>
```

Otherwise, how would MkDocs know about it?

[1]: http://www.mkdocs.org/user-guide/configuration/#site_url

**Comment by waylan** [comment]
This was addressed in #1039.

**Comment by Vinno97** [comment]
Whoops. This is embarrasing :/
Had that line quoted out..

Sorry for the useless issue, but thank you for your swift response
